lib/webrtc/ZombieRTCPeerConnection.js

```diff
@@ -142,6 +142,16 @@ ZombieRTCPeerConnection.prototype.addStream = function (stream) {
   this._localStreams.push(stream);
 };
 
+ZombieRTCPeerConnection.prototype.removeStream = function (stream) {
+  this._localStreams.push(stream);
```

Comment:
This line seems a little strange to me.  Is there a good reason to push the stream into the local streams before looking to remove it?